frontend/src/components/user/OpportunityCompetitionStart.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import { useFeatureFlags } from "src/hooks/useFeatureFlags";
+import { useUser } from "src/services/auth/useUser";
+import { Competition } from "src/types/competitionsResponseTypes";
+
+import StartApplicationModal from "src/components/workspace/StartApplicationModal";
+
+export const OpportunityCompetitionStart = ({
+  competitions,
+}: {
+  competitions: [Competition];
+}) => {
+  const { user } = useUser();
+  const { checkFeatureFlag } = useFeatureFlags();
+
+  const openCompetitions = competitionData({ competitions });
+
+  if (
+    !openCompetitions.length ||
+    !user?.token ||
```

Comment:
[suggestion] if there's no user token, wouldn't we want to show an unauthenticated message? Or is that still in the works?

Reply:
For now we just don't show the button. I learned yesterday it will show for anon and will ask people to login.